client/ayon_core/hosts/houdini/plugins/create/create_review.py

```diff
@@ -31,11 +32,9 @@ def create(self, product_name, instance_data, pre_create_data):
 
         frame_range = hou.playbar.frameRange()
 
-        filepath = "{root}/{product_name}/{product_name}.$F4.{ext}".format(
-            root=hou.text.expandString("$HIP/pyblish"),
-            # keep dynamic link to product name
-            product_name="`chs(\"AYON_productName\")`",
-            ext=pre_create_data.get("image_format") or "png"
+        filepath = self.staging_dir.format(
+            product={"name": "`chs(\"AYON_productName\")`"},
```

Comment:
Is there a reason this is the only one (that I could quickly see) that uses channel reference? I suppose this was just missed?

Reply:
yes, it's still the only one. I'll update the rest too.